src/modules/review/review.service.js

```diff
@@ -0,0 +1,65 @@
+/** load required packages */
+
+/** load peer modules and services */
+const Review = require('./review.schema');
+
+/**
+ * ReviewService operates on the data layer of the application, and performs *all* db operations.
+ *
+ * ReviewService is consumed not only by ReviewController, but also by controllers of other modules.
+ */
+class ReviewService {
+  /**
+   * Fetch all Review of a specified courser with id 
+   * @returns Array<Review> list of Reviews in the system of a course
+   */
+  static async getCourseReviews(id) {
+    const reviewList = await Review.find({course:id});
+    return reviewList;
+  }
+
+  /**
+   * Create new Review entry
+   * @param reviewObj object on which a new review will be created.
+   * @returns Review object newly created
+   */
+  static async createNewReview(reviewObj) {
+    const newReview = await Review.create(reviewObj);
+    return newReview;
+  }
+  /**
+   * Update a Review document
+   * @param  updatedReviewObj 
+   * @returns review object which is updated
+   */
+  static async updateReview(updatedReviewObj) {
+    const updatedReview = await Review.findByIdAndUpdate(
+      updatedReviewObj._id,
+      updatedReviewObj
+    );
```

Comment:
https://mongoosejs.com/docs/api.html#model_Model.findByIdAndUpdate
By default, findByIdAndUpdate() returns the document as it was before update was applied.
```suggestion
    const updatedReview = await Review.findByIdAndUpdate(
      updatedReviewObj._id,
      updatedReviewObj,
      {
          new : true
      }
    );
```